package.json

```diff
@@ -47,6 +47,7 @@
     "@dimensiondev/holoflows-kit": "0.8.0-20201215084554-2f9e757",
     "@dimensiondev/kit": "0.0.0-20201126061915-b58b8f0",
     "@dimensiondev/matrix-js-sdk-type": "=1.0.0-20200731085604-dc2de92",
+    "@dimensiondev/metamask-extension-provider": "^3.0.0-20210125122006-86e9bae",
```

Comment:
Hello! Please rebase this and add this dependency to `/packages/maskbook/package.json`, thanks